src/api/raw.rs

```diff
@@ -553,6 +577,48 @@ impl RawApi {
         Ok(req.body(data).context(ErrorKind::RequestBuild)?)
     }
 
+    /// Get a pod logs
+    pub fn log(&self, name: &str, lp: &LogParams) -> Result<http::Request<Vec<u8>>> {
```

Comment:
I would put special case `impl RawApi` in its own `impl` block or file, but other than that this setup is good.